__tests__/app.test.js

```diff
@@ -0,0 +1,74 @@
+const request = require('supertest');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data');
+const db = require('../db/connection');
+
+afterAll(() => {
+    if (db.end) db.end();
+  });
+
+beforeEach(() => seed(data));
+
+describe('app()', () => {
+     describe('GET /api/topics', () => {
+        test('status:200, responds with an array of topic objects', () => {
+            return request(app)
+              .get('/api/topics')
+              .expect(200)
+              .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toBeInstanceOf(Array);
+            });
+        });
+        test(`status:200, responds with array of topic objects: properites 
+                slug: string, description: string`, () => {
+            return request(app)
+              .get('/api/topics')
+              .expect(200)
+              .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toBeInstanceOf(Array);
+                topics.forEach((topic) => {
+                            expect(topic).toEqual(
+                                    expect.objectContaining({
+                                        slug: expect.any(String),
+                                        description: expect.any(String),
+                                    })
+                            );
+                        });
+              });
+        });
+        test('status:200, responds with an array of length 3 topic objects', () => {
+            return request(app)
+              .get('/api/topics')
+              .expect(200)
+              .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toBeInstanceOf(Array);
```

Comment:
repeated assertion

Reply:
removed the repeated assertion, asserting instance of array